apps/api/src/applicant/ienapplicant.service.ts

```diff
@@ -535,4 +538,32 @@ export class IENApplicantService {
     };
     return this.ienapplicantJobRepository.findAndCount(query);
   }
+
+  async assignApplicant(id: string, employee: EmployeeRO) {
+    const applicant = await this.ienapplicantRepository.findOne(id);
+
+    const found = applicant?.recruiters?.some(e => e.id === employee.id);
+    if (found) return;
+
+    const haPcn = await this.haPcnRepository.findOne({ title: employee.organization });
+    if (!haPcn) {
+      throw new BadRequestException(`User doesn't belong to a health authority`);
```

Comment:
small nit - param is called employee but we're throwing an error calling it a user, also maybe add a few comments just to showcase what's happening in this fn

Reply:
emplyee.organization column is not related to haPcn by a foreign key constraint, so it could be any string.